### DISCUSSION TITLE
Entering watched state crashes Kodi Matrix 19.4

### DISCUSSION THREAD

**Discussion by azido** [croneter/PlexKodiConnect#1838] [Q&A] [answered]
Hey there,

when ever i watched an episode/movie/whatever or mark one of these as watched in Kodi, Kodi immediately stops working and reboots.
After the reboot, the state is changed, though. Happened with beta version of PKC, so i did a fresh libreelec install and added PKC stable instead. Same trouble. Does NOT happen if i mark watched videos back to unwatched state.

Any hint on this?

**Comment by croneter** [comment]
Log files would help: https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by azido** [comment]
Yeah, never mind. I guess it is "multiple paths" related. Having more than one directory for the same kind of media might be a valuable choice (e.g. using dirs HD-MOVIES on one drive and HD-MOVIES-2 on another, both feeding content into same library) with standalone plex (never had trouble) but can possibly lead to trouble in combination with kodi :/ Need to check my setup first, clean up things and try again. If prob still exists i'm back with a log.

**Comment by azido** [comment, marked as answer]
Got it sorted. Was indeed a "dual-path" problem. Stuck on direct path setting for another 2 days and finally got it running with several movie libraries.. damned, that was hard to configure :) Does not really work without manually editing sources.xml and passwords.xml, though.

Have another problem with music playlists, something like "ordinal not in range (128) at initial sync, might be because of plex smart playlists not pointing to something useful atm.. dunno, but i get through that too for sure ;O

**Reply by InvisibleKnight** [reply]
I'm having the same exact issue, may I ask you to walk me through some of your configurations? I could not figure out how to solve this problem.

**Comment by croneter** [comment]
@azido Could you be so kind and share your configuration that helped you solve this issue? @InvisibleKnight would be glad for you help 😉
